### PR TITLE
Adding Jenkins slave node setup

### PR DESCRIPTION
**Pull request by melissa-kam**
Instead of updating the current Jenkins swarm script to support Xenial as well as Trusty, I switched to a python script that uses the Jenkins API to accommodate for the CIT Jenkins restrictions. I also left out the server deployment since that's being converted to groovy scripts.

Connects https://github.com/rcbops/u-suk-dev/issues/940

### CODE REVIEW

File: playbooks/setup-jenkins-slave.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+- hosts: cloud_instances
+  tasks:
+    - name: Install apt packages
+      apt:
+        pkg: "{{ item }}"
+        state: installed
+        update_cache: yes
+      with_items:
+        - git
+        - default-jre
```
Comment (mattt416) [inline playbooks/setup-jenkins-slave.yml]: Is this needed?

**Comment by mattt416** [comment]
Ha, guess we didn't think about this.  Anyway, one inline nit but otherwise this LGTM!

File: playbooks/setup-jenkins-slave.yml (hunk 1 above)
Comment (melissa-kam) [inline playbooks/setup-jenkins-slave.yml]: If you mean default-jre specifically, it's needed to run the Jenkins slave jar since it's Java based.

Comment (mattt416) [inline playbooks/setup-jenkins-slave.yml]: 👍

**Pull request merged**